src/Apps/Notifications/NotificationsApp.tsx

```diff
@@ -24,48 +24,50 @@ const NotificationsApp: React.FC<NotificationsAppProps> = ({ me }) => {
   )
 
   return (
-    <NotificationsContextProvider>
-      <MetaTags title="Notifications | Artsy" pathname="/notifications" />
+    <FullBleed>
```

Comment:
This must be added here, even if it is already declared as a design in the routes file. It is required so that the page really has the full width.